.buildkite/pipeline.yml

```diff
@@ -0,0 +1,14 @@
+steps:
+  - label: ":golang: Build"
+    command: "cd src && go build ."
+    plugins:
+      - golang#v2.0.0:
+          version: 1.21
+          import: github.com/buildkite/agent
+
+  - label: ":test-analytics: Test"
+    command: "cd src && go test ./..."
+    plugins:
+      - golang#v2.0.0:
+          version: 1.21
+          import: github.com/buildkite/agent
```

Comment:
This is a bit of a nitpick, but this file is missing a trailing newline which isn't _really_ a problem, it just clutters future diffs a bit.

Perhaps we can add a https://editorconfig.org/ file as most of the common editors support that out of the box

Reply:
The result of some editor gymnastics 😀 Yep, will throw editorconfig in